users/vieting/experiments/switchboard/ctc/feat/experiments.py

```diff
@@ -617,16 +617,20 @@ def run_scf_audio_perturbation_from_checkpoint():
     }
 
     perturbation_args = [
-        {"codecs": [{"encoding": "ULAW", "prob": 0.3, "minimum": 250, "maximum": 260}]},
-        {"codecs": [{"encoding": "ULAW", "prob": 0.7, "minimum": 250, "maximum": 260}]},
-        {"codecs": [{"encoding": "ULAW", "prob": 0.7, "minimum": 240, "maximum": 270}]},
-        {"codecs": [{"encoding": "ULAW", "prob": 1, "minimum": 250, "maximum": 260}]},
+        {"codecs": [{"encoding": "ULAW", "prob": 0.3, "minimum": 1, "maximum": 10, "default": None}]},
+        {"codecs": [{"encoding": "ULAW", "prob": 0.7, "minimum": 1, "maximum": 10, "default": None}]},
+        {"codecs": [{"encoding": "ULAW", "prob": 0.7, "minimum": 1, "maximum": 20, "default": None}]},
+        {"codecs": [{"encoding": "ULAW", "prob": 1, "minimum": 1, "maximum": 10, "default": None}]},
         {"tempo": {"prob": 0.3, "minimum": 0.83, "maximum": 1.17}},
         {"tempo": {"prob": 0.8, "minimum": 0.83, "maximum": 1.17}},
         {"tempo": {"prob": 0.3, "minimum": 0.7, "maximum": 1.3}},
         {"tempo": {"prob": 0.8, "minimum": 0.7, "maximum": 1.3}},
         {"tempo": {"prob": 1, "minimum": 0.83, "maximum": 1.17}},
         {"tempo": {"prob": 1, "minimum": 0.7, "maximum": 1.3}},
+        {"pitch": {"prob": 0.3, "minimum": -2, "maximum": 2}},
+        {"pitch": {"prob": 0.3, "minimum": -3, "maximum": 3}},
+        {"pitch": {"prob": 0.7, "minimum": -2, "maximum": 2}},
+        {"pitch": {"prob": 0.7, "minimum": -3, "maximum": 3}},
```

Comment:
Can you shortly explain how you chose the min and max values?

Reply:
mostly based on listening to sound samples. Everything outside of this range sounded unrealistic. If the bigger ranges result in better WERs I can add more ranges later